packages/jetbrains-plugin/src/main/kotlin/com/mongodb/jbplugin/editor/EditorToolbarDecorator.kt

```diff
@@ -83,16 +83,16 @@ class EditorToolbarDecorator(
     override fun editorCreated(event: EditorFactoryEvent) {
         editor = event.editor
 
-        toolbar = MdbJavaEditorToolbar(
-            editor.project!!,
-            coroutineScope,
-            onConnected = this::onConnected,
-            onDisconnected = this::onDisconnected,
-            onDatabaseSelected = this::onDatabaseSelected,
-            onDatabaseUnselected = this::onDatabaseUnselected
-        )
-
         editor.project?.let { project ->
+            toolbar = MdbJavaEditorToolbar(
+                project,
+                coroutineScope,
+                onConnected = this::onConnected,
+                onDisconnected = this::onDisconnected,
+                onDatabaseSelected = this::onDatabaseSelected,
+                onDatabaseUnselected = this::onDatabaseUnselected
+            )
```

Comment:
This I moved inside only because I noticed that earlier we were asserting that editor.project is not null and I saw NullPointerException a couple of times in my IDE. Still not sure if it was because of this (probably not) but this change felt harmless. Let me know though if its not.

Reply:
That sounds good! It might throw an exception if you are opening a file not in a project (like a scratch file or something) so this makes sure doesn't happen.